### ISSUE TITLE
NOT A BUG BUT A QUESTION

### ISSUE THREAD

**Issue by dittmarconsulting** [panz3r/react-native-flipper-databases#302]
Thank you for this plugin.

Everything works fine and I can see the Realm DB in Flipper.

I have a question about the query syntax for Realm DB. I can query a table in JS like that

```js
let contacts = realm.objects('Contacts');
```

but what would the syntax be for the Flipper input field to query the `Contacts`?

<img width="567" alt="Screen Shot 2022-03-18 at 6 22 06 pm" src="https://user-images.githubusercontent.com/19203201/158955265-4fe880d1-f5c9-4eb3-9b41-03e1404c0256.png">

**Comment by panz3r** [comment]
Hi @dittmarconsulting ,
Thanks for your feedback!

Currently making SQL queries is NOT supported when using `react-native-flipper-databases` because the supported DB are all NoSQL ones.

The best way for you to see the data contained in your `contacts` collection is using the `Data` tab and selecting the collection in the select field on the right.

**Comment by dittmarconsulting** [comment]
Hi @panz3r,

Thank you so much for the fast reply and your suggestion. Much appreciated.

I have another question regarding the `Data` tab. In our Realm DB we store objects as you can see in the screenshot below.

![Screen Shot 2022-03-19 at 10 52 23 am](https://user-images.githubusercontent.com/19203201/159097937-467da928-7a88-4a4d-a92f-30625be32e39.png)

I would like to change the source code so that I can display a `JSON` string of these objects. Can you please give me a pointer in which file I could change that?

I would also like to suggest that as a future feature for those who might need that. 

Thanks in advance.

**Comment by dittmarconsulting** [comment]
Hi @panz3r ,

All good. I found it. I changed the return value in file `node_modules/react-native-flipper-databases/src/drivers/realm.ts` like that and it worked like a treat

```js
function getCellValue(row: Record<string, unknown>, columnName: string): unknown {
  const cellValue = row[columnName];
  // return typeof cellValue === 'object' ? cellValue?.toString() : cellValue;
  return typeof cellValue === 'object' ? JSON.stringify(cellValue, null, 4) : cellValue;
}
```

**Comment by panz3r** [comment]
Hi @dittmarconsulting ,

Thanks for your feedback and for the suggestion, that was actually a bug in how the value was returned for `object` cells.

I fixed it and released a new version (`v2.3.1`) to address the issue.
